src/boot_O2_g3/boot_main.c

```diff
@@ -2,6 +2,7 @@
 #include "global.h"
 #include "stack.h"
 #include "stackcheck.h"
+#include "z64thread.h"
 
```

Comment:
```suggestion
#include "stack.h"
#include "stackcheck.h"
#include "z64thread.h"
```
Is `"global.h"` still needed?

Reply:
Sadly yes, because of function declarations and variables